nservicebus/vscode.md

```diff
@@ -0,0 +1,263 @@
+            "Website"
+        ]
+    }
+]
+```
+
+### Complete example
+
+This snippet shows an entire launch.json file with **Sales** and **Billing** endpoints plus a **Website** project all in a directory named `src` beneath the project root:
+
+```json
+{
+   "version": "0.2.0",
+   "configurations": [
+        {
+            "name": "Billing",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build",
+            "program": "${workspaceFolder}/src/Billing/bin/Debug/netcoreapp2.1/Billing.dll",
+            "args": [],
+            "cwd": "${workspaceFolder}/src/Billing",
+            "stopAtEntry": true,
+            "console": "externalTerminal"
+        },
+        {
+            "name": "Sales",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build",
+            "program": "${workspaceFolder}/src/Sales/bin/Debug/netcoreapp2.1/Sales.dll",
+            "args": [],
+            "cwd": "${workspaceFolder}/src/Sales",
+            "stopAtEntry": false,
+            "console": "externalTerminal"
+        },
+        {
+            "name": "Website",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build",
+            "program": "${workspaceFolder}/src/Website/bin/Debug/netcoreapp2.1/Website.dll",
+            "args": [],
+            "cwd": "${workspaceFolder}/src/Website",
+            "stopAtEntry": false,
+            "internalConsoleOptions": "openOnSessionStart",
+            "launchBrowser": {
+                "enabled": true,
+                "args": "${auto-detect-url}",
+                "windows": {
+                    "command": "cmd.exe",
+                    "args": "/C start ${auto-detect-url}"
+                },
+                "osx": {
+                    "command": "open"
+                },
+                "linux": {
+                    "command": "xdg-open"
+                }
+            },
+            "env": {
+                "ASPNETCORE_ENVIRONMENT": "Development"
+            },
+            "sourceFileMap": {
+                "/Views": "${workspaceFolder}/src/Website/Views"
+            }
+        },
+        {
+            "name": ".NET Core Attach",
+            "type": "coreclr",
+            "request": "attach",
+            "processId": "${command:pickProcess}"
+        }
+    ,],
+    "compounds": [
+        {
+            "name": "Debug All",
+            "configurations": [
+                "Billing",
+                "Sales",
+                "Website",
+            ]
+        }
+    ]
+}
+```
+
+## More information
+
+See the following articles for more information on debugging with VSCode:
+
+* [OmniSharp: Configuring launch.json for C# debugging](https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md)
+* [Visual Studio Code: Integrate with External Tools via Tasks](https://code.visualstudio.com/docs/editor/tasks)
```

Comment:
This was already linked in the document. Do we want to duplicate it here?

Reply:
Yeah, I like that they're summarily mentioned here together.